index.html

```diff
@@ -61,19 +61,16 @@ <h2>Welcome to ROS Index</h2>
  <hr>
   <div class="row">
   	<h2>Active Distributions</h2>
-    <p>
-      <a href="https://wiki.ros.org/melodic">ROS Melodic</a>
-    </p>
```

Comment:
Melodic is still supported until 2023.  Did you mean to take this out?

Reply:
Woops, no. I was too far ahead thinking about Jammy migrations etc.